.ci/aarch64/kubernetes/init.sh

```diff
@@ -8,5 +8,12 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-k8s_version=$(kubectl version | base64 | tr -d '\n')
-network_plugin_config="https://cloud.weave.works/k8s/net?k8s-version=${k8s_version}"
+network_plugin_config_file="${SCRIPT_PATH}/../../.ci/${arch}/kubernetes/kube-flannel.yml"
+
+# original flannel has limitation and request for memory, it may cause OOM on AArch64
+sudo -E curl -fsL "https://raw.githubusercontent.com/coreos/flannel/bc79dd1505b0c8681ece4de4c0d86c5cd2643275/Documentation/kube-flannel.yml" -o $network_plugin_config_file
+# install yq if not exist
+${SCRIPT_PATH}/../../.ci/install_yq.sh
+sudo -E ${GOPATH}/bin/yq d -i -d5 $network_plugin_config_file spec.template.spec.containers[*].resources.*.memory > /dev/null
```

Comment:
- Strictly, this should probably be `GOBIN` but we don't seem to use that in the CI.

  /cc @chavafg - I wonder if we want to set `GOBIN` here: https://github.com/kata-containers/tests/blob/master/.ci/lib.sh#L27?

- Could you quote that `spec.template.spec...` path as I get nervous when I see commands running as `root` using unquoted asterisks :smile:

Reply:
Hi~ Let me make this whole thing a variable, maybe it will more ease you. ;) 🤩